src/main/java/org/roaringbitmap/RoaringArray.java

```diff
@@ -578,4 +578,30 @@ protected int size() {
   public void writeExternal(ObjectOutput out) throws IOException {
     serialize(out);
   }
+
+  /**
+   * Gets the first value in the array
+   * @return te first value in the array
+   */
+  public int first() {
+    if(size == 0) {
+      return 0;
```

Comment:
Yes, Collections::max was the first that came to my mind.
I thought about 0 as a possible value as I know that when using bitmaps as index for lines of DB tables, those lines are refered by their index as in an array - starting from 0.
Anyway, I like it very much now :)